app/components/status-badge.jsx

```diff
@@ -30,17 +30,21 @@ const useStyles = createUseStyles(theme => ({
     lineHeight: '1.375rem',
   },
   number: {
-    fontWeight: '600',
+    fontWeight: '800',
```

Comment:
should be 600. In figma you can Control+Click on the number and see the CSS of it.